src/main/scala/org/scalajs/tools/tsimporter/Trees.scala

```diff
@@ -36,6 +36,7 @@ object Trees {
     case object Static extends Modifier
     case object ReadOnly extends Modifier
     case object Const extends Modifier
+    case object Override extends Modifier
```

Comment:
This is not the right place to declare `Override`. `Trees` represents TypeScript trees, and hence `Trees.Modifier` represents TypeScript modifies, not Scala ones.

Either you add another concept of Scala modifiers in `Definitions.scala`, or simply add a boolean `needsOverride` in `MethodSymbol`.

Reply:
Heh, this is what I initially did, but then over-engineered it. Changed to use `needsOverride` in `MethodSymbol`.